static/src/javascripts/projects/common/modules/experiments/tests/contributions-membership-epic-brexit.js

```diff
@@ -35,22 +35,25 @@ define([
 
     return function () {
 
-        this.id = 'ContributionsMembershipEpic';
-        this.start = '2016-11-03';
+        this.id = 'ContributionsMembershipEpicBrexit';
+        this.start = '2016-11-04';
         this.expiry = '2016-11-07';
         this.author = 'Jonathan Rankin';
-        this.description = 'Find the optimal way of offering Contributions along side Membership in the Epic component';
+        this.description = 'Find the optimal way of offering Contributions along side Membership in the Epic component on stories about Brexit';
         this.showForSensitive = true;
-        this.audience = 0.15;
-        this.audienceOffset = 0.71;
+        this.audience = 1;
+        this.audienceOffset = 0;
         this.successMeasure = 'Impressions to number of contributions/supporter signups';
         this.audienceCriteria = 'All users in US and UK';
         this.dataLinkNames = '';
-        this.idealOutcome = 'One of the 3 variants proves to be a clear winner';
+        this.idealOutcome = 'One of the 2 variants proves to be a clear winner';
         this.canRun = function () {
             var userHasNeverContributed = !cookies.get('gu.contributions.contrib-timestamp');
             var worksWellWithPageTemplate = (config.page.contentType === 'Article'); // may render badly on other types
-            return userHasNeverContributed && commercialFeatures.canReasonablyAskForMoney && worksWellWithPageTemplate;
+            var keywords = config.page.keywordIds.split(',');
+            var nonKeywordTagIds = config.page.nonKeywordTagIds.split(',');
+            var isAboutBrexit = (keywords.indexOf('politics/eu-referendum') != -1) && (nonKeywordTagIds.indexOf('tone/news') != -1);
```

Comment:
In fact you could have:

    var hasEuReferendumTag = config.page.keywordIds.split(',').indexOf(...) !== -1

if you thought it was clearer.